penner.js

```diff
@@ -77,17 +77,12 @@
       return -c / 2 * (Math.cos(Math.PI * t / d) - 1) + b;
     },
     easeInExpo: function(t, b, c, d) {
-      var _ref;
-      return (_ref = t === 0) != null ? _ref : {
-        b: c * Math.pow(2, 10 * (t / d - 1)) + b
-      };
+      return (t==0) ? b : c * Math.pow(2, 10 * (t/d - 1)) + b;
```

Comment:
i'm not sure what this is fixing. if you want to correct a mistake, please:

1. edit penner.coffee
2. run `grunt` to compile it down to js
3. make a pr with penner.coffee, penner.js, and penner.min.js